### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by AminaBuhari**
Page titles:- No issues found
Text heading:- No issues found.
Color contrast:- Change button text color to black to increase contrast
Resize:- No issues found.
Interaction:- No issues found.
Moving contents:- No issues found.
Multimedia: No issues found.
The basic structure of the page :- No issues found.
Image text alternative:- Added text to the hamburger icon.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,16 +5,29 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio Website</title>
+    <title>Portfolio Project Website</title>
```
Comment (micheaol) [inline index.html:8]: * Poor titles:
  * *Microverse project | Your portfolio.*
  * *Portifolio project*
* Better page titles:
  * *Mark's portifolio*
  * *Mark full-stack webdeveloper*
 Kindly make the necessary adjustments to the title.

File: style.css [hunk 2]
```diff
@@ -189,7 +189,7 @@ ul {
   background-color: #36b37e;
   border-radius: 5px;
   border-style: none;
-  color: white;
+  color: black;
   font-size: 17px;
```
Comment (micheaol) [inline style.css:193]: **Contrast ratio ("color contrast")**:

* The page has a minimum contrast ratio of at least 4.5:1 for normal-size text.
Please check the report [HERE](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2022-02-22-at-20-27-40-935?failing) according to the project requirement.

File: index.html [hunk 3]
```diff
@@ -5,16 +5,29 @@
...
+        }
+        gtag("js", new Date());
+
+        gtag("config", "G-29VQ5BWQNZ");
+    </script>
 </head>
 
 <body>
     <header>
         <nav class="menu-bar">
```
Comment (micheaol) [inline index.html:27]: **Forms**:

* Every form control has a label associated with it using 'label', 'for', and 'id' according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @RaoAkif , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 2 above)
Comment (AminaBuhari) [reply style.css:193]: I changed the color from white to black. The black color increased the contrast ratio thus making the website accessibility to  100%.

**Pull request merged**